pkg/regexp/regexp.go

```diff
@@ -22,7 +22,10 @@ func Delayed(val string) Regexp {
 	}
 	if precompile {
 		re.regexp = regexp.MustCompile(re.val)
+	} else {
+		re.once = &sync.Once{}
 	}
+
```

Comment:
Having multiple `Regexp` values pointing to a shared `once` doesn’t make sense.

Instead, make `type Regexp` a `*struct`, and avoid the copy that way.

Reply:
Filed #1636 .